packages/expo-image/ios/EXImageView.m

```diff
@@ -13,10 +17,14 @@
 @interface EXImageView ()
 
 @property (nonatomic, weak) RCTBridge *bridge;
+@property (nonatomic, strong) SDAnimatedImageView *imageView;
 @property (nonatomic, strong) NSDictionary *source;
 @property (nonatomic, assign) RCTResizeMode resizeMode;
 @property (nonatomic, assign) BOOL needsReload;
 @property (nonatomic, assign) CGSize intrinsicContentSize;
+@property (nonatomic, assign) EXImageCornerRadii cornerRadii;
+@property (nonatomic, assign) EXImageBorders borders;
+@property (nonatomic, strong) NSDictionary<NSString *, CALayer *> *borderLayers;
```

Comment:
Is there a reason not to use `NSMutableDictionary`?

Reply:
Do you mean for just `borderLayers`, or also for `cornerRadii` and `borders` ?

Reply:
Updated as part of the CornerRadii/Borders refactoring 👍